src/domain/profile/ProfileUtil.ts

```diff
@@ -26,8 +28,17 @@ export const normalizeProfileChild = (
   };
 };
 
-export const getProjectsFromProfileQuery = (children: Children) => {
-  return sortedUniq(
-    children?.edges?.map((child) => child?.node?.project.year).sort()
-  );
+export const getProjectsFromProfileQuery = (children: Children): Project[] => {
+  const empty: Project = {
```

Comment:
This is a way to prevent TS errors (since the generated types are all with `| undefined`). So many things I don't know how to do.